src/User_Command.php

```diff
@@ -429,7 +429,12 @@ public function create( $args, $assoc_args ) {
 		if ( is_multisite() ) {
 			$result = wpmu_validate_user_signup( $user->user_login, $user->user_email );
 			if ( is_wp_error( $result['errors'] ) && ! empty( $result['errors']->errors ) ) {
-				WP_CLI::error( $result['errors'] );
+				$message = $result['errors']->get_error_message();
+				if ( false !== stripos( $message, '</strong>' ) ) {
+					$exploded = explode( '</strong>', $message );
+					$message  = trim( wp_strip_all_tags( $exploded[1] ) );
+				}
```

Comment:
This feels a bit too clever and does not explain what it does.

Why not simplify to:

```suggestion
				$message  = trim( wp_strip_all_tags( $message ) );
				$message  = str_replace( 'Error: ', '', $message );
```

Reply:
Hmm then I'd rather lean towards not removing the double "Error: ". I don't think we do that anywhere else, so I don't see why we should do it here.

And doing `stripos` and `explode` is very fragile as the strings can change in WordPress. So I prefer keeping it simple.

But curious to hear what others think.

Reply:
We have used similar in Plugin command. https://github.com/wp-cli/extension-command/blob/5619a55fd279ce016a299b898bbe8eedb2aecb53/src/Plugin_Command.php#L366 But it seems we have replaced english `Error:` here also.

Reply:
May be we should use simple `$message  = trim( wp_strip_all_tags( $message ) ); $message  = str_replace( 'Error: ', '', $message );` and not worry about the string in Behat test.

Reply:
Yeah WP-CLI is not usually translated